src/components/structures/UserSettings.js

```diff
@@ -379,19 +395,42 @@ module.exports = React.createClass({
                             Disable inline URL previews by default
                         </label>
                     </div>
+                    { settingsLabels.map( setting => {
```

Comment:
oh. duh.

(although it rather raises the question of why this code is here at all, if we didn't notice it being fucked/bitrotten. Still, a separate question)